mappings/net/minecraft/entity/ai/goal/Goal.mapping

```diff
@@ -1,5 +1,18 @@
 CLASS net/minecraft/class_1352 net/minecraft/entity/ai/goal/Goal
 	FIELD field_6451 controls Ljava/util/EnumSet;
+	METHOD method_38846 shouldRunEveryTick ()Z
+		COMMENT {@return if the goal should run every tick or not}
+		COMMENT
+		COMMENT <p>This returns {@code false} by default. If this returns false,
+		COMMENT the goal will tick once after the entity is spawned, and will tick
+		COMMENT every other ticks.
+		COMMENT
+		COMMENT @see #getTickCount(int)
+	METHOD method_38847 getTickCount (I)I
+		COMMENT {@return how many times a goal can tick in the given {@param ticks} at most}
+		ARG 1 ticks
+	METHOD method_38848 divideByTickInterval (I)I
```

Comment:
Maybe `toGoalTicks(int serverTicks)`?

Reply:
Hmm. For modding perspective there is really no reason to call this (`getTickCount` should be called instead), because `divideByTickInterval` is just an implementation of `getTickCount` for not-always-ticking goals. Better names, perhaps?

Reply:
> For modding perspective there is really no reason to call this

But afai see in enigma, mojang is calling this rampantly in its codebase. The other one cannot be called in a static context (they have a lot of static utility methods, I guess?), and often times whether a goal ticks bitickly or tickly is already statically known.